testsuite/features/secondary/srv_scc_user_credentials.feature

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: SCC user credentials in the Setup Wizard
+The purpose of this feature is not the same that the feature in core, 
```

Comment:
No :-) . First, a feature is a feature of the **product**. A feature is not about testing.
We test product features.

Second, use standard syntax for Feature header.
```
Feature: SCC user credentials in the Setup Wizard
  As a systems administrator
  In order to manage organization credentials at SUSE Customer Service
  I want to create, edit, and delete credentials
```

Reply:
Fully agree :)